nautilus_core/model/src/orders/base.rs

```diff
@@ -656,6 +656,85 @@ impl OrderCore {
     pub fn init_event(&self) -> Option<OrderEventAny> {
         self.events.first().cloned()
     }
+
+    pub fn generate_order_submitted_event(
```

Comment:
As discussed, I think there is a better location for these helpers.

Not where and how we'll generate these events in the production code, and currently these are just for a test.

Reply:
Refactored here https://github.com/nautechsystems/nautilus_trader/pull/1650/commits/234a7355a9b82d2e96f5e7f8254eb2a613d91720